Task#1 correct

andremare opened the pull request:
1. new method renderUsersQualities
2. randerPhrase corresponds to the function
3. use %10

andremare committed:
Task#1 correct
1. new method renderUsersQualities
2. randerPhrase corresponds to the function
3. use %10

AndreiSachuk commented on src/components/users.jsx:
```diff
@@ -9,14 +9,20 @@ const Users = () => {
     }
 
-    const renderPhrase = () => {
+    const renderUsersQualities = (user) => {
```
сюда лучше квалитис передавать, а не юзера

AndreiSachuk commented on src/components/users.jsx:
```diff
@@ -9,14 +9,20 @@ const Users = () => {
+    }
+
+    const renderUsersTable = () => {
         return users.map(user => (
```
тут было бы гуд сделать деструктуризацию

andremare replied:
Как это сделать внутри метода map?

andremare committed:
Task#1 correct-2
1. destructuring of renderUsersTable()
2. renderPhrase() takes qualities instead of users

andremare commented:
Внесены все корректировки
https://github.com/andremare/course-project_fast-company/commit/aa1927c4268198d6bad53ed3c04a7da1189dfa08
